tools/test_runner.js

```diff
@@ -80,21 +79,22 @@ Runner.prototype.checkSkipModule = function() {
 
 
 Runner.prototype.run = function() {
-  if (this.attr.skip && (this.attr.skip.indexOf('all') >= 0 ||
-      this.attr.skip.indexOf(this.driver.os) >= 0)) {
+  if (skip = this.test['skip']) {
```

Comment:
I hope to declare `skip`  variable for the function scope, or this `skip`, which someone might use this name on later, let him/her be very hard to seek to find where it is used.